src/components/flash/__spec__.js

```diff
@@ -148,28 +148,28 @@ describe('Flash', () => {
     describe("if there is an event", () => {
       it('calls preventDefault on it', () => {
         let spy = jasmine.createSpy('preventDefault');
-        defaultInstance.toggleDialog(null, { preventDefault: spy });
+        defaultInstance.toggleDialog(null)({ preventDefault: spy });
         expect(spy).toHaveBeenCalled();
       });
     });
 
     it('sets the state to the opposite of what it was', () => {
       defaultInstance.setState({ dialogs: { foo: false }});
-      defaultInstance.toggleDialog('foo');
+      defaultInstance.toggleDialog('foo')();
```

Comment:
This is part of the refactor to remove `bind` being called on each render. `toggleDialog` now accepts an argument and returns a function that can be used by a click handler.